Update quickstart.md (#1631)
jupyter-book/mystmd: docs/quickstart.md

description: Get up and running with the MyST (Markedly Structured Text) command line interface. MyST is designed to create publication-quality documents written entirely in Markdown.
---

> These quickstart tutorials are here to get you up and running on your local computer 👩‍💻. Thus, you will then be able to:

Reviewer: ```suggestion
These quickstart tutorials are here to get you up and running on your local computer 👩‍💻. You will learn how to:
```

      url: https://mystmd.org/guide
  domains: []
```

> % 2024-11-09 | @sugan-reden : I added the following note because the .yaml file I got with the quickstart was different than the one shown above. Feel free to edit/delete it (along with this comment) if not approved.
> 

Reviewer: ```suggestion
```
